lib/workers/repository/init/vulnerability.ts

```diff
@@ -207,7 +207,7 @@ export async function detectVulnerabilityAlerts(
           // TODO: types (#22198)
           const allowedVersions =
             datasource === PypiDatasource.id
-              ? `==${val.firstPatchedVersion!}`
+              ? `>=${val.firstPatchedVersion!}`
```

Comment:
What about the line below it too?

Reply:
Hmm, then overall this should work:

```suggestion
          const allowedVersions =
            datasource === MavenDatasource.id
              ? `[${val.firstPatchedVersion!},]`
              : `>=${val.firstPatchedVersion!}`;
```

I'm goint to run it on a larger test repo to verify, just in case.

Reply:
**Update**: Looks good -> https://github.com/renovate-demo/29586-vulndemo/pulls

The only issue is that this breaks `nuget` security updates since https://github.com/renovatebot/renovate/pull/26150 is missing. Hence, `allowedVersions` for nuget would need to stay at `val.firstPatchedVersion!` as an interim solution.

I'd prefer doing the change to all managers in a separate PR. This PR related to PyPI could be merged for now and would do no harm but if you prefer only one change, feel free to close here.